site/src/server.js

```diff
@@ -4,7 +4,7 @@ import sirv from 'sirv';
 import * as sapper from '@sapper/server';
 import { sanitize_user, authenticate } from './utils/auth';
 
-const { PORT = 3000 } = process.env;
+const port = process.env.port || 3000;
```

Comment:
This should be `process.env.PORT`, no?

This still seems like a really weird error and a really weird fix.

Reply:
yes, thanks!

agreed it's a bit weird, but it fixes it and I'm not sure I really want to spend more time on it given all the other stuff to work on